lib/DataFetcher.dart

```diff
@@ -155,8 +160,22 @@ class DataFetcher {
     ];
   }
 
-  static Future<bool> uploadGrades(String filePath) {
-    return Future.value(true); // TODO send to server
+  static uploadGrades(String filePath) async {
+
+    Dio dio = new Dio();
+
+    final String serverEndPoint = "http://localhost:3000/filePath"; // Temporary
```

Comment:
This should be using DataFetcher's API field that has the proper base endpoint URL.